index.rst

```diff
@@ -6,6 +6,7 @@ Contents:
 .. toctree::
    :maxdepth: 2
 
+   user/introduction
```

Comment:
this page is not for users but for developers, move it there and move it below in the index